content/en/product_analytics/segmentation/_index.md

```diff
@@ -9,7 +9,147 @@ further_reading:
 
 ## Overview
 
-Segmentation is a feature in Product Analytics that allows you to analyze and understand specific groups or segments of your user base. By segmenting your users based on various characteristics and behaviors, you can uncover valuable insights, identify trends, and make data-driven decisions to optimize your product and user experience. For example, you can segment users by purchase amount, active users in a given country, users in a trial, or users in a trial who then converted to a paying organization.
+Analyze and understand specific groups or segments of your user base using the [User Profile][3] and the [Segments][1] features. 
+
+The User Profiles page lists your available product users. You can **list**, **search**, **sort**, and **filter** on these users to create a list of these key users. You can use the [user profile attributes](#user-profile-attributes) to filter on the users you want, or [create a segment](#build-a-segment), to better analyze and understand their usage of your product. 
+
+The segments page includes a list of all of the segments you have created --- you can also create a segment from this page. Segmenting helps you to focus on some of your users based on various characteristics and behaviors in order to uncover valuable insights, identify trends, and make data-driven decisions that will optimize your product and user experience. For example, you can segment users by purchase amount, active users in a given country, users in a trial, or users in a trial who then converted to a paying organization. 
+
+Follow the content list here to navigate to: 
+
+- **[User Profiles and their usage:](#user-profiles)**
+  - [User profile attributes](#user-profile-attributes)
+  - [How profiles are sourced](#how-profiles-are-sourced)
+  - [Navigating the user profile page](#navigating-the-user-profile-page)
+
+- **Segments - how to build and use them:**
+  - [Building a segment using your profiles](#build-a-segment)
+  - [Using the segments you created](#leverage-segments-across-product-analytics)
+
+
```

Comment:
```suggestion

```
The right-nav already provides this, automatically. We tend to not duplicate this except in extreme circumstances (which this doesn't appear to be)